packages/database/src/migrations/20210602030420-AddFijiCOVIDVaccinatedByFacilityOverlays-modifies-data.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const createMapOverlay = doseNum => ({
+  id: `FJ_COVID_TRACKING_Dose_${doseNum}_Facility`,
+  name: `COVID-19 Vaccine Dose ${doseNum} (Facility)`,
+  userGroup: 'COVID-19',
+  dataElementCode: doseNum === 1 ? 'COVIDVac4' : 'COVIDVac8',
+  measureBuilderConfig: {
+    aggregations: [
+      {
+        type: 'FINAL_EACH_DAY',
+      },
+    ],
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+    },
+  },
+  measureBuilder: 'sumAllPerOrgUnit',
+  presentationOptions: {
+    scaleType: 'neutral',
+    displayType: 'spectrum',
+    scaleBounds: {
+      left: {
+        max: 0,
+        min: 0,
```

Comment:
from card [#2906](https://github.com/beyondessential/tupaia-backlog/issues/2906) it said the lower bound should be 1